homeassistant/components/fan/insteon_local.py

```diff
@@ -88,11 +88,11 @@ def setup_fan(device_id, name, insteonhub, hass, add_devices_callback):
         configurator.request_done(request_id)
         _LOGGER.info("Device configuration done!")
 
-    conf_fans = config_from_file(hass.config.path(INSTEON_LOCAL_FANS_CONF))
+    conf_fans = load_json(hass.config.path(INSTEON_LOCAL_FANS_CONF))
     if device_id not in conf_fans:
         conf_fans[device_id] = name
 
-    if not config_from_file(
+    if not save_json(
```

Comment:
When will save json return `False`? It seems to raise an error in two cases.

Reply:
You are right, i looked at the new save_json and saw it returns False at the end, but didn't question the code itself.

There were several components with some custom error logging if save failed, guess we can just get rid of all those occurrences.

Reply:
Yeah, since @balloob thought it better to blow up in that case when reviewing #10216 .